keycloak/lib/stacks/vpc.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import {
+  IpAddresses, Peer, Port, SecurityGroup, SelectedSubnets, SubnetType, Vpc,
+} from 'aws-cdk-lib/aws-ec2';
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class VpcStack extends Stack {
+  public readonly vpc: Vpc;
+
+  public readonly keyCloaksecurityGroup: SecurityGroup
+
+  public readonly rdsDbSecurityGroup: SecurityGroup
+
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id);
+    this.vpc = new Vpc(this, 'KeycloakVpc', {
```

Comment:
may be add the `maxAzs` configuration as well?

Reply:
Done!